test/central_planner_test.py

```diff
@@ -343,6 +343,7 @@ def test_do_not_lock_resources_when_not_ready(self):
         self.assertEqual('C', self.sch.get_work(worker='Y')['task_id'])
 
     def test_lock_resources_when_one_of_multiple_workers_is_ready(self):
+        self.sch.get_work(worker='X')
```

Comment:
Can you add a comment for these new lines why you are doing them?

Even with the context of this PR, I'm not certain. I suppose it's for making the worker "active"?